src/Abp.AspNetCore/AspNetCore/AbpApplicationBuilderExtensions.cs

```diff
@@ -93,10 +93,16 @@ public static void UseAbpRequestLocalization(this IApplicationBuilder app, Actio
             using (var languageManager = iocResolver.ResolveAsDisposable<ILanguageManager>())
             {
                 var supportedCultures = languageManager.Object
-                    .GetLanguages()
+                    .GetActiveLanguages()
                     .Select(l => CultureInfo.GetCultureInfo(l.Name))
                     .ToArray();
 
+                if (iocResolver.IsRegistered<ILogger<RequestLocalizationOptions>>())
+                {
+                    using var logger = iocResolver.ResolveAsDisposable<ILogger<RequestLocalizationOptions>>();
```

Comment:
Let’s put this into a block.

```csharp
using (var logger = ....)
{
    logger.Object.LogInformation(...);
}
```

Reply:
Nothing will change, it will be disposed after if block

Reply:
Probably it is bette to be consistent with other places in the code base which we always write `using` with block